include/eredis_sub.hrl

```diff
@@ -1,34 +1,34 @@
 -record(state, {
-          host :: string() | undefined,
```

Comment:
If the lines end up too far to the right, it's possible to line break before `{`. Here or in other places.

```Erlang
-record(state,
        {host :: string() | undefined,
         port :: ...